docs/typescript.mdx

```diff
@@ -241,8 +241,8 @@ type Theme = {
 export default styled as CreateStyled<Theme>
 
 // You can also create themed versions of all the other theme helpers and hooks
-const { ThemeProvider, withTheme, useTheme } = emotionTheming as emotionTheming.EmotionTheming<Theme>
-export { ThemeProvider, withTheme, useTheme }
+const { ThemeProvider, useTheme } = { ThemeProvider, useTheme } as EmotionTheming<Theme>
```

Comment:
The `ThemeProvider` and `useTheme` are lacking as imports, rendering the documentation a little incomplete.

Reply:
good point, gonna fix together with other docs stuff 👍

Reply:
If we do the global theme I think a lot of the work around this area will be wasted. Just something to keep in mind